torch/csrc/jit/import.cpp

```diff
@@ -165,16 +179,18 @@ void ScriptModuleDeserializer::convertModule(
 
 void import_ir_module(
     ModuleLookup module_lookup,
-    std::istream& in) {
+    std::istream& in,
+    DeviceLookup device_lookup) {
   ScriptModuleDeserializer deserializer(&in);
-  deserializer.deserialize(module_lookup);
+  deserializer.deserialize(module_lookup, device_lookup);
 }
 
 void import_ir_module(
     ModuleLookup module_lookup,
-    const std::string& filename) {
+    const std::string& filename,
+    DeviceLookup device_lookup) {
   ScriptModuleDeserializer deserializer(filename);
-  deserializer.deserialize(module_lookup);
+  deserializer.deserialize(module_lookup, device_lookup);
 }
 
 std::shared_ptr<script::Module> load(std::istream& in) {
```

Comment:
shall we expose device_lookup in C++ API too? might be useful

Reply:
Yep, I kept that in mind. My plan is do it in another diff.